components/prism-fsharp.js

```diff
@@ -0,0 +1,16 @@
+Prism.languages.fsharp = Prism.languages.extend('clike', {
+	'comment': [
+		{
+			pattern: /(^|[^\\])\(\*[\w\W]*?\*\)/g,
+			lookbehind: true
+		},
+		{
+			pattern: /(^|[^\\:])\/\/.*?(\r?\n|$)/g,
```

Comment:
We most certainly don’t. Think of CSS styling that adds a background too. This should be converted to a lookahead. Thanks for catching that Golmote!